scripts/ros-generate-recipes.sh

```diff
@@ -47,17 +47,14 @@ if [ -z "$SUPERFLORE_GEN_OE_RECIPES" ]; then
 fi
 
 ROS_DISTRO=$1
-# ROS_VERSION and ROS_PYTHON_VERSION must be in the environment as they appear in "conditional" attributes. Keep this block in
-# sync with the one in ros-generate-cache.sh .
+# Keep this block in sync with the one in ros-generate-cache.sh .
 case $ROS_DISTRO in
     "melodic"|"noetic")
         export ROS_VERSION="1"
```

Comment:
> ROS_VERSION (not exported)

It's still being exported in this script.

Reply:
Yes, but it doesn't need to be exported I believe.

Reply:
So shouldn't `export` be dropped?